vnext/Microsoft.ReactNative.SharedManaged/JSValueReaderGenerator.cs

```diff
@@ -83,16 +94,16 @@ static JSValueReaderGenerator()
       s_genericMethods = new Lazy<IReadOnlyDictionary<Type, SortedList<Type, MethodInfo>>>(() =>
       {
         var genericMethods =
-          from pair in s_allMethods.Value
-      where pair.Value.IsGenericMethod
-      let type = pair.Key
-      let keyType = type.GetTypeInfo().IsGenericType ? type.GetGenericTypeDefinition()
-        : type.IsArray ? typeof(Array)
-        : throw new InvalidOperationException($"Unsupported argument type {type}")
-      group pair by keyType into g
-      select new KeyValuePair<Type, SortedList<Type, MethodInfo>>(
-        g.Key, new SortedList<Type, MethodInfo>(
-          g.ToDictionary(p => p.Key, p => p.Value), GenericTypeComparer.Default));
+          from pair in s_allMethods.Value.Union(s_codeGenerateGenericExtensionMethods)
```

Comment:
>s_codeGenerateGenericExtensionMethods [](start = 48, length = 37)

So, for now generic methods are still based on reflection, right?

Reply:
The generic methods need overload resolution and the existing implementation performs the overload resolutoin accross all possible registered assemblies in the app, so there is no way to do that at codegen, unless we are okay with a breaking change and say overload resolution happens on a per-assemlby only.